chart/openfaas/values.yaml

```diff
@@ -85,6 +85,17 @@ ingress:
   tls:
     # Secrets must be manually created in the namespace.
 
+# faas-idler configuration
+faasIdler:
+  replicas: 1
+  image: openfaas/faas-idler:0.1.7
+  gatewayUrl: http://gateway.openfaas:8080/
```

Comment:
I'd rather see this string derived in the chart in-line.

Reply:
You mean from the CLI argument?

Reply:
Hard-coding this string is wrong, it has to be templated to include the namespace we are installing into. I.e. if you installed into `openfaas-prod` this would be: `gateway.openfaas-prod:8080` not `gateway.openfaas:8080`